### PR TITLE
completed html-forms exercise

### PR DESCRIPTION
**Pull request by dcatan89**
Kapture 2021-11-30 at 14 39 20

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <div>Why is coding cool?</div>
+    <select name="Options">
+      <option value="Select an option">Select an option</option>
+      <option value="Because its fun">It is fun</option>
+      <option value="Beause it's powerful">Because it's powerful</option>
+      <option value="Because it's interesting">Becuase it's interesting</option>
+    </select>
+    <div>Favorite Language (choose one)</div>
+    <input type="radio" name="coding language" value="HTML" />
+    HTML
```
Comment (codingcodymiller) [inline html-forms/index.html]: This text should be in a `label` element.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  </head>
+  <h1>Coding Form!</h1>
+
+  <form action="">
+    <label for="email"> Email </label>
+    <div>
+      <input type="email" name="email" id="email" required />
+    </div>
+
+    <div>Why is coding cool?</div>
```
Comment (codingcodymiller) [inline html-forms/index.html]: This text should be in a `label` element.

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      <option value="Select an option">Select an option</option>
+      <option value="Because its fun">It is fun</option>
+      <option value="Beause it's powerful">Because it's powerful</option>
+      <option value="Because it's interesting">Becuase it's interesting</option>
+    </select>
+    <div>Favorite Language (choose one)</div>
+    <input type="radio" name="coding language" value="HTML" />
+    HTML
+    <input type="radio" name="coding language" value="CSS" />
+    CSS
```
Comment (codingcodymiller) [inline html-forms/index.html]: This text should be in a `label` element.

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    <input type="radio" name="coding language" value="JavaScript" />
+    JavaScript
```
Comment (codingcodymiller) [inline html-forms/index.html]: This text should be in a `label` element.

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+    <div>Would you like to sign up for more information?</div>
+    <input type="checkbox" name="sign me up" value="Sign me up" />
+    Sign Me Up!
```
Comment (codingcodymiller) [inline html-forms/index.html]: This text should be in a `label` element.

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+    <div>Comments</div>
```
Comment (codingcodymiller) [inline html-forms/index.html]: This text should be in a `label` element.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-forms/index.html (hunk 6 above)
Comment (dcatan89) [reply html-forms/index.html]: Issue updated with label elements

File: html-forms/index.html (hunk 5 above)
Comment (dcatan89) [reply html-forms/index.html]: Issue updated with label elements

File: html-forms/index.html (hunk 4 above)
Comment (dcatan89) [reply html-forms/index.html]: Issue updated with label elements

File: html-forms/index.html (hunk 3 above)
Comment (dcatan89) [reply html-forms/index.html]: Issue updated with label elements

File: html-forms/index.html (hunk 2 above)
Comment (dcatan89) [reply html-forms/index.html]: Issue updated with label elements

File: html-forms/index.html (hunk 1 above)
Comment (dcatan89) [reply html-forms/index.html]: Issue updated with label elements

File: html-forms/index.html [hunk 7]
```diff
@@ -31,18 +35,19 @@ <h1>Coding Form!</h1>
...
 
     <p>Would you like to sign up for more information?</p>
     <label for="sign me up">
       <input type="checkbox" name="sign me up" value="Sign me up" />
       Sign Me Up!
     </label>
-    <div>
+
+
       <h4>Comments</h4>
```
Comment (codingcodymiller) [inline html-forms/index.html]: This should be a `label` element.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-forms/index.html (hunk 7 above)
Comment (dcatan89) [reply html-forms/index.html]: Added label element to it

File: html-forms/index.html [hunk 8]
```diff
@@ -3,51 +3,53 @@
...
-    <label>
-      <div>Why is coding cool?</div>
-      <select name="Options">
-        <option value="Select an option">Select an option</option>
-        <option value="Because its fun">It is fun</option>
-        <option value="Beause it's powerful">Because it's powerful</option>
-        <option value="Because it's interesting">Becuase it's interesting</option>
-     </select>
-    </label>
+      <label>
```
Comment (codingcodymiller) [inline html-forms/index.html]: This `label` element is unnecessary.

File: html-forms/index.html [hunk 9]
```diff
@@ -3,51 +3,53 @@
...
-    </label>
 
-
-      <h4>Comments</h4>
-      <textarea type="text" name="comment" cals="20" rows="8"> </textarea>
-  <div>
-        <input type="submit" name="submit" value="Submit" />
-      </div>
-  </form>
+        <h4>Comments</h4>
```
Comment (codingcodymiller) [inline html-forms/index.html:48]: This should be a `label` element.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-forms/index.html (hunk 8 above)
Comment (dcatan89) [reply html-forms/index.html]: label removed

File: html-forms/index.html (hunk 9 above)
Comment (dcatan89) [reply html-forms/index.html:48]: label element added

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**